tests/test_thirdparty.py

```diff
@@ -3,11 +3,24 @@
 ###########################################################################
 import os
 import sys
-import warnings
+from pathlib import Path
 
 import pytest
 
 
+def test_module_paths():
+    "Third party binary modules have meaningful __file__ attributes"
+    import PIL
+    from PIL import _imaging
+
+    # iOS and Android both play shenanigans with binary locations.
+    # Make sure the __file__ attribute on the binary module reflects
+    # the actual location in the file system. The base PIL module is
+    # pure Python; the binary module for PIL._imaging should be in
+    # the same folder.
```

Comment:
The module doesn't have to *be* in the same folder, its `__file__` attribute just has to appear that way.

```suggestion
    # its package's location in the file system. The base PIL module is
    # pure Python; the binary module for PIL._imaging should have a 
    # __file__ attribute pointing to the same folder.
```